Content.Server/Access/Systems/AccessOverriderSystem.cs

```diff
@@ -197,7 +201,7 @@ private void TryWriteToTargetAccessReaderId(EntityUid uid,
         if (!PrivilegedIdIsAuthorized(uid, component))
             return;
 
-        if (!_interactionSystem.InRangeUnobstructed(uid, component.TargetAccessReaderId))
+        if (!_tagSystem.HasTag(player, BypassInteractionRangeChecksTag) && !_interactionSystem.InRangeUnobstructed(uid, component.TargetAccessReaderId))
```

Comment:
```suggestion
        if (!_interactionSystem.InRangeUnobstructed(player, component.TargetAccessReaderId))
```
there we go, this should fix the bug
revert the other changes in this file, please

can you also add 
`inHandsOnly: true` to the `ActivatableUI` component in the prototype? That prevents some weird behaviour where you can use it from a distance

Reply:
add to the access config proto?

Reply:
done